src/components/Fund/FundBudgets/FundBudgets.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { Accordion } from '@folio/stripes/components';
+
+import ConnectionListing from '../../ConnectionListing';
+
+const budgetColumns = ['name', 'allocated', 'unavailable', 'available'];
+
+const FundBudgets = ({
+  addBudgetButton,
+  budgets,
+  budgetStatus,
+  currency,
+  labelId,
+  openBudget,
+  sectionId,
+}) => (
+  <Accordion
+    displayWhenOpen={addBudgetButton(budgetStatus, budgets.length)}
+    id={sectionId}
```

Comment:
it doesn't look like the best solution to move accordions down, into the components. IMHO, they describe the page layout, and passing `sectionId` like this is quite ugly. @NikitaSedyx what do you think?

Reply:
But in this case, I need to use `displayWhenOpen` which depends on `budgets.length`